<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>1017</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_chooseFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_chosenFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chosen File:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_maxChans">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>240</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Chans:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loadedChans">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loaded Chans:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_calStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cal status</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_refChannel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_manualCal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual Cal Ref Channel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_align">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Align To Ref</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calibrate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate Following Plan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plotEnergy">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot Energy Hist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>741</width>
     <height>931</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
